src/components/Map/components/LegendEntry.styled.ts

```diff
@@ -1,5 +1,17 @@
 import styled, { css } from 'styled-components';
 
+export const LabelLegendInputLabelWrapper = styled.div`
+  margin-left: 4px;
+`;
+export const LabelLegendInputLabel = styled.div`
+  display: flex;
+`;
+export const LabelLegendDescription = styled.div`
+  margin-left: 8px;
+  font-size: 13px;
+  line-height: 1.5;
```

Comment:
1.5 what ?

Reply:
1.5 of Font-size (It is an proportional value) 

From MDN > https://developer.mozilla.org/fr/docs/Web/CSS/line-height
`Unitless values: use this number multiplied by the element's font size`